pkg/capabilities/consensus/ocr3/ocr3_consensus-schema.json

```diff
@@ -0,0 +1,131 @@
+        },
+        "Err": {
+          "type": "boolean"
+        }
+      },
+      "additionalProperties": false,
+      "required": ["WorkflowExecutionID", "Value", "Err"]
+    },
+    "FeedValue": {
+      "type": "object",
+      "properties": {
+        "deviation": {
+          "type": "string",
+          "description": "The deviation that is required to generate a new report. Expressed as a percentage. For example, 0.01 is 1% deviation."
+        },
+        "heartbeat": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The interval in seconds after which a new report is generated, regardless of whether any deviations have occurred. New reports reset the timer."
+        },
+        "remappedID": {
+          "type": ["string", "null"],
+          "description": "An optional remapped ID for the feed."
+        }
+      },
+      "additionalProperties": false,
+      "required": ["deviation", "heartbeat"]
+    }
+  },
+  "properties": {
+    "config": {
+      "properties": {
+        "aggregation_method": {
+          "type": "string",
+          "enum": ["data_feeds"]
+        },
+        "aggregation_config": {
+          "type": "object",
+          "properties": {
+            "allowedPartialStaleness": {
+              "type": "string",
+              "description": "Allowed partial staleness as a number between 0 and 1."
+            },
+            "feeds": {
+              "type": "object",
+              "propertyNames": {
+                "$ref": "../../triggers/streams/streams_trigger-schema.json#/$defs/feedId"
+              },
+              "additionalProperties": {
+                "$ref": "#/$defs/FeedValue"
+              }
+            }
+          },
+          "additionalProperties": false,
+          "required": ["allowedPartialStaleness", "feeds"]
+        },
+        "encoder": {
+          "type": "string",
+          "enum": ["EVM"]
+        },
+        "encoder_config": {
+          "type": "object",
+          "properties": {
+            "abi": {
+              "type": "string",
+              "description": "The ABI for report encoding.",
+              "examples": [
+                "(bytes32 FeedID, uint224 Price, uint32 Timestamp)[] Reports"
+              ]
+            }
+          },
+          "additionalProperties": false,
+          "required": ["abi"]
+        },
+        "report_id": {
+          "type": "string",
+          "pattern": "^[a-f0-9]{4}$",
+          "examples": ["0x0001"]
+        }
+      },
+      "additionalProperties": false,
+      "type": "object",
+      "required": [
+        "aggregation_method",
+        "aggregation_config",
+        "encoder",
+        "encoder_config",
+        "report_id"
+      ]
+    },
+    "inputs": {
+      "properties": {
+        "observations": {
+          "type": "array",
+          "items": {
+            "$ref": "../../triggers/streams/streams_trigger-schema.json#/$defs/Feed"
+          }
+        }
```

Comment:
Can someone confirm if this is right?  When @DeividasK and I where working on it, this felt natural, but looking closer at the code I think it accepts an array of arrays of feed, not just an array of feeds